### PR TITLE
Align content to header

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?
Align content to header.

#### What problem is this solving?
Reduce content page padding, so the content follows header alignment

#### How should this be manually tested?
[Access this workspace](https://alignheader--storecomponents.myvtex.com/)

#### Screenshots or example usage
New alignment
captura de tela de 2018-10-31 05-55-09

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by matheusps** [comment]
The footer padding is defined inside of dreamstore-footer, and is changed in [THIS PR](https://github.com/vtex-apps/dreamstore-footer/pull/10)

File: react/store-global.css [hunk 1]
```diff
@@ -43,39 +43,33 @@ body {
 .vtex-page-padding {
   padding-left: 0;
   padding-right: 0;
+  margin-right: auto;
+  margin-left: auto;
```
Comment (vitoriaHeliane) [inline react/store-global.css]: Use tachyons

File: react/store-global.css [hunk 2]
```diff
@@ -43,39 +43,33 @@ body {
...
+@media only screen and (min-width: 64em) {
   .vtex-page-padding {
-    padding-left: 15.625rem;
-    padding-right: 15.625rem;
+    width: 90%;
   }
 }
 
-@media screen and (min-width: 120em) {
+@media screen and (min-width: 80em) {
```
Comment (luciannojunior) [inline react/store-global.css]: Tachyons in this class isn't also achievable?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Fix paddings to match header
```
Comment (luciannojunior) [inline CHANGELOG.md]: Finish it with a .

**Pull request merged**